packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -313,7 +317,7 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
   /**
    * Centers Pan and resets the zoom for stack viewport.
    */
-  public resetCamera: (resetPan?: boolean, resetZoom?: boolean) => boolean;
+  public resetCamera: (options?) => boolean;
```

Comment:
Could we get a type declaration here on options - it is hard to know what values to provide, and it makes getting the right splelllnig in the call.